process/block/shardblock.go

```diff
@@ -1873,24 +1875,35 @@ func (sp *shardProcessor) createMiniBlocks(haveTime func() bool, randomness []by
 	var miniBlocks block.MiniBlockSlice
 
 	if sp.accountsDB[state.UserAccountsState].JournalLen() != 0 {
-		log.Error("shardProcessor.createMiniBlocks", "error", process.ErrAccountStateDirty,
-			"stack", string(sp.accountsDB[state.UserAccountsState].GetStackDebugFirstEntry()),
-		)
+		log.Error("shardProcessor.createMiniBlocks",
+			"error", process.ErrAccountStateDirty,
+			"stack", string(sp.accountsDB[state.UserAccountsState].GetStackDebugFirstEntry()))
 
+		interMBs := sp.txCoordinator.CreatePostProcessMiniBlocks()
+		if len(interMBs) > 0 {
+			miniBlocks = append(miniBlocks, interMBs...)
+		}
+
+		log.Debug("creating mini blocks has been finished", "num miniblocks", len(miniBlocks))
 		return &block.Body{MiniBlocks: miniBlocks}, nil
 	}
 
 	if !haveTime() {
 		log.Debug("shardProcessor.createMiniBlocks", "error", process.ErrTimeIsOut)
+
+		interMBs := sp.txCoordinator.CreatePostProcessMiniBlocks()
+		if len(interMBs) > 0 {
+			miniBlocks = append(miniBlocks, interMBs...)
+		}
+
+		log.Debug("creating mini blocks has been finished", "num miniblocks", len(miniBlocks))
 		return &block.Body{MiniBlocks: miniBlocks}, nil
 	}
 
 	startTime := time.Now()
 	mbsToMe, numTxs, numMetaHeaders, err := sp.createAndProcessMiniBlocksDstMe(haveTime)
 	elapsedTime := time.Since(startTime)
-	log.Debug("elapsed time to create mbs to me",
-		"time [s]", elapsedTime,
-	)
+	log.Debug("elapsed time to create mbs to me", "time [s]", elapsedTime)
```

Comment:
maybe remove [s] ? the time will contain the measuring unit by default

Reply:
done

Reply:
Done